source/guide_tiddlywiki5.rst

```diff
@@ -0,0 +1,187 @@
+.. highlight:: console
+.. author:: GodMod <godmod@eqdkp-plus.eu>
+
+.. tag:: wiki
+.. tag:: web
+.. tag:: lang-javascript
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/tiddlywiki.svg
+      :align: center
+
+############
+TiddlyWiki 5
+############
+
+.. tag_list::
+
+TiddlyWiki_ is a personal wiki and a non-linear notebook for organising and sharing complex information.
+
+License
+=======
+
+TiddlyWiki_ is published under a `permissive BSD 3-Clause License`_.
+
+Prerequisites
+=============
+
+If you want to use TiddlyWiki with your own domain you need to setup your domain first:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Step 1
```

Comment:
Please replace numbered steps with a description of what is going on. The numbers do not add any information and make it cumbersome to edit the guide later.